release/_publishing.sh

```diff
@@ -144,7 +144,7 @@ function upload_release {
 
 	for file in * .*
```

Comment:
@pgydev, why don't you just restrict the files from here? I mean, exclude the content of folder `"${_BUILD_DIR}/boms"`?

Reply:
@natocesarrego Do you mean that the for loop shouldn't go through the content of the `"${_BUILD_DIR}/boms"`. Wouldn't that be more complicated?

Reply:
> Do you mean that the for loop shouldn't go through the content of the "${_BUILD_DIR}/boms"
Yes

> Wouldn't that be more complicated?
Depends of the way we'll implement it :slightly_smiling_face: